addons/website/views/website_templates.xml

```diff
@@ -1004,7 +1004,7 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
 
 <template id="website_search_box" name="Website Searchbox">
     <div t-attf-class="input-group #{_classes}" role="search">
-        <input type="text" name="search" class="search-query form-control oe_search_box" placeholder="Search..." t-att-value="search"/>
+        <input type="text" name="search" class="search-query form-control oe_search_box" t-att-placeholder="placeholder if placeholder else 'Search...'" t-att-value="search"/>
```

Comment:
This is breaking translations... :)

Reply:
Indeed. Will look at it asap :) . First fwd-port .